cluster-up/cluster/kind/common.sh

```diff
@@ -95,7 +98,37 @@ function _configure_network() {
 function prepare_workers() {
     # appending eventual workers to the yaml
     for ((n=0;n<$(($KUBEVIRT_NUM_NODES-1));n++)); do
-        echo "- role: worker" >> ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/kind.yaml
+        cat << EOF >> ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/kind.yaml
+- role: worker
+  kubeadmConfigPatches:
+  - |
+    kind: JoinConfiguration
```

Comment:
This should be in Cluster config, not per node: https://github.com/kubernetes-sigs/kind/blob/master/pkg/apis/config/v1alpha3/types.go#L41

Reply:
Not a change for this PR, but we need to resolve it on kubevirtci.

Reply:
Cannot be cluster config, kubevirt e2e tests expect this to not be activated are master, at qemu provides is only activated at workers, nodes.sh

Reply:
Yeah, I think I have to switch API version of that type. Was doing some more debugging. Will post all the changes so we can review it on Monday. On my test PR, sriov lane passes again